### ISSUE TITLE
lost compitability with firefox for android?

### ISSUE THREAD

**Issue by jorjio** [yuvadm/free-haaretz-ff#6]
Hi,
previously i installed the extension easily on firefox for android (lolipop) .
somehow this ability got lost when i changed my phone (running android marshmallow now), 
other github XPI files are installed properly.
any ideas?

MANY thanks for this wonderful extension!

**Comment by yuvadm** [comment]
yeah, i just rebuilt the extension with the new toolchain for v0.5. probably forgot to add some flag. looking into this

**Comment by yuvadm** [comment]
@jorjio not sure why `jpm-mobile` doesn't support building an XPI like `jpm` does. Do you have any experience with this stuff?

**Comment by jorjio** [comment]
sorry, no idea about these stuff really...
thanks..

**Comment by jorjio** [comment]
sorry, no idea about these stuff really...
thanks..
